source/declarators.tex

```diff
@@ -2388,11 +2388,11 @@
 \indextext{initialization|(}
 
 \pnum
-A declarator can specify an initial value for the
+An \grammarterm{init-declarator} can specify an initial value for the
```

Comment:
Hmm. It might also be better to put some distance between the initialization rules, which generally have an initializer and some target entity, and the special case of initializing in a declaration. Perhaps this paragraph should start by talking about initialization in general, and then say that one of the places this happens is when a declarator is followed by an initializer (and other places include initialization of function parameters from arguments and initialization of return values).